homeassistant/components/homematicip_cloud/switch.py

```diff
@@ -156,31 +156,17 @@ class HomematicipMultiSwitch(HomematicipGenericEntity, SwitchEntity):
 
     def __init__(self, hap: HomematicipHAP, device, channel: int) -> None:
         """Initialize the multi switch device."""
-        self.channel = channel
-        super().__init__(hap, device, f"Channel{channel}")
-
-    @property
-    def name(self) -> str:
-        """Return the name of the multi switch channel."""
-        label = self._get_label_by_channel(self.channel)
-        if label:
-            return label
-        return super().name
-
-    @property
-    def unique_id(self) -> str:
-        """Return a unique ID."""
-        return f"{self.__class__.__name__}_{self.post}_{self._device.id}"
```

Comment:
I'm referring to the removal of the unique id property here in the switch platform. This had post so doesn't seem to match the parent class unique id that has channel.

Reply:
Aaaah. This is the MultiSwitch implementation.
Old implementation used the post (line 160) f"Channel{channel}
New implementation uses the channel f"{self.__class__.__name__}_Channel{self._channel}_{self._device.id}"

The result is an identical unique id.

Reply:
Ok.